examples/hello-monai-bundle/config/config_fed_server.json

```diff
@@ -0,0 +1,64 @@
+{
+  "format_version": 2,
+
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
+      "name": "PTFileModelPersistor",
+      "args": {
+      "model": {
+            "path": "monai.networks.nets.unet.UNet",
+            "args": {
+              "dimensions": 3,
+              "in_channels": 1,
+              "out_channels": 2,
+              "channels": [16, 32, 64, 128, 256],
+              "strides": [2, 2, 2, 2],
+              "num_res_units": 2,
+              "norm": "batch"
+            }
+          }
+      }
+    },
+    {
+      "id": "shareable_generator",
+      "name": "FullModelShareableGenerator",
+      "args": {}
+    },
+    {
+      "id": "aggregator",
+      "name": "AccumulateWeightedAggregator",
+      "args": {
+        "aggregation_weights": {
+          "org1-a": 1.0,
+          "org1-b": 0.5
+        },
+        "expected_data_kind": "WEIGHTS"
+      }
+    }
```

Comment:
can you update this part to

might also want to update the hello-monai example

```

    {
      "id": "aggregator",
      "path": "nvflare.app_common.aggregators.intime_accumulate_model_aggregator.InTimeAccumulateWeightedAggregator",
      "args": {
        "expected_data_kind": "WEIGHTS"
      }
    }

```